Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
Appreciate it's an important part of your mechanism here, but I'm quite uncomfortable with adding the entire `HOMEBREW_PREFIX` bin back into the `$PATH`. It'd be fine for producing bottles but on local machines where folks have other bits and pieces installed the chances of scooping something out of the `$PATH` that shouldn't be scooped is very high :confused:.

Would using the path to Postgresql's `libexec/bin` not do the trick at all? I presume not, if you've chosen to use `HOMEBREW_PREFIX` here.